django/src/rdwatch/tasks/__init__.py

```diff
@@ -206,5 +215,15 @@ def get_siteobservations_images(
 @shared_task
 def delete_temp_model_runs_task() -> None:
     """Delete all model runs that are due to be deleted."""
-    print('Test')
-    # TODO: implement this
+    model_runs_to_delete = HyperParameters.objects.alias(
+        delete_at=ExpressionWrapper(
+            F('created') + F('expiration_time'), output_field=DateTimeField()
+        )
+    ).filter(delete_at__lte=timezone.now())
```

Comment:
What is the behavior of this if the expiration_time is null/blank?  Should there be a specific value that prevents deletion (-1 or 0 or something else)?

Reply:
Yeah, I didn't actually configure a null row in the table to verify, but just wanted to double-check.  I know that with a time of 0 and setting the update rate to 1 minute it would immediately delete it.  The explicit thing should make it easier to reason about what's going on when someone else looks at it in the future, thanks.